libiocage/lib/Jail.py

```diff
@@ -18,6 +18,49 @@
 
 
 class Jail:
+    """
+    Iocage unit orchestrates a jail's configuration and manages state
+
+    Jails are represented as zfs dataset `zpool/iocage/jails/<NAME>`
+
+    Directory Structure:
+
+        zpool/iocage/jails/<NAME>: The jail's dataset containing it's
+            configuration and root directory. Iocage legacy used to store
```

Comment:
lowercase `iocage`, we can drop and root directory. It's not a directory, but a dataset. And isn't terribly important in this description. I think we should drop the JSON being preferred, and make the language stronger. It's the only supported mechanism. There are a lot of new things and properties, and this will help ease the migrations to it if we only support that mechanism.